README.md

```diff
@@ -3,6 +3,33 @@
 [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=products/micro-integrator)](https://wso2.org/jenkins/job/products/job/micro-integrator/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# WSO2 Micro Integrator
+
+## Introduction
+
+In recent times we have witnessed the growth of approaches like Continuous Integration and Continuous Deployment (CI/CD), which has subsequently led to the rise of microservices. Microservices are designed to start up very quickly, perform a specific task, and then shut down.
```

Comment:
This is good. But maybe we should mention that the MSA became more popular due to the popularity of cloud-native technologies and the assistance it provides for agile development.